datus-mysql/datus_mysql/connector.py

```
@@ -264,10 +264,21 @@ def get_schema(
+            SELECT
+                COLUMN_NAME as Field,
+                COLUMN_TYPE as Type,
+                IS_NULLABLE as `Null`,
+                COLUMN_KEY as `Key`,
+                COLUMN_DEFAULT as `Default`,
+                COLUMN_COMMENT as Comment
+            FROM INFORMATION_SCHEMA.COLUMNS
+            WHERE TABLE_SCHEMA = '{database_name}'
+              AND TABLE_NAME = '{table_name}'
+            ORDER BY ORDINAL_POSITION
+        """
```

Ruff (0.14.8)

Possible SQL injection vector through string-based query construction

(S608)